test/test.js

```diff
@@ -126,6 +126,22 @@ test('linting of incorrect Markdown file fails with absolute path', async t => {
   }
 });
 
+test('linting of unreadable Markdown file fails', async t => {
+  const unreadablePath = '/tmp/unreadable.md';
```

Comment:
Could you do relative to __dirname? I don't want to reach out like this as some systems may treat that as malware, etc..

Reply:
Also, it breaks on Windows. :)

Reply:
Eurghhh. This was causing race issues with the other tests, that's why I went with `/tmp` originally. Need to figure out somewhere it doesn't break the glob tests.

Reply:
You can give it a non-standard extension? .mdtest or something?